src/Elrond/NetworkConfig.h

```diff
@@ -0,0 +1,49 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include <string>
+
+namespace TW::Elrond {
+
+/// A "NetworkConfig" object holds the network parameters relevent to creating transactions (e.g. minimum gas limit, minimum gas price).
+class NetworkConfig {
+    /// The following fields can (should) be fetched from https://api.elrond.com/network/config.
+    /// However, a "NetworkConfig" object is initialized with proper default values for Elrond Mainnet (as of December 2021).
+    std::string chainId;
+    uint32_t gasPerDataByte;
+    uint32_t minGasLimit;
+    uint64_t minGasPrice;
+
+    /// GasSchedule entries of interest (only one at this moment), according to: https://github.com/ElrondNetwork/elrond-config-testnet/blob/master/gasSchedules.
```

Comment:
maybe provide the URL for config mainnet? since in the above comment you mention mainnet

Reply:
Fixed :rocket: